src/main/main.ts

```diff
@@ -673,10 +685,24 @@ async function initializeStandalone(): Promise<void> {
 
           break;
         } catch (error) {
-          console.error(
-            `Unexpected error occurred during download / extract snapshot. ${error}` +
-              `path: ${path}`
-          );
+          errorOccurred = true;
+          const errorMessage = `Unexpected error occurred during download / extract snapshot.\n${error}`;
```

Comment:
`path` is deleted. is this correct change?

Reply:
Yes it is. The `path` used as argument of `snapshot.downloadMetadata()` and `snapshot.downloadSnapshot()`. And that used to make the `downloadPath` variable inside of these methods. Each methods throw new exceptions now(`DownloadSnapshotFailedError`, `DownloadSnapshotMetadataFailedError`) and these exceptions error messages contains the `downloadPath`.